### ISSUE TITLE
Redesigning right sidebar of validation mission screens

### ISSUE THREAD

**Issue by jonfroehlich** [ProjectSidewalk/SidewalkWebpage#2260]
While overseeing with my son's Kindergarten today, I mocked up some more informative and gamified versions of the right sidebar in the validation mission screen. I added validation rate information (to help users compare their validation behavior to others) and some gamifiction ideas. What do you think @gari01234 and @misaugstad?

![image](https://user-images.githubusercontent.com/1621749/94173439-cede5880-fe48-11ea-90a9-e950feddfdc3.png)

![image](https://user-images.githubusercontent.com/1621749/94173462-d6056680-fe48-11ea-8138-02c031e06881.png)

![image](https://user-images.githubusercontent.com/1621749/94173510-e3225580-fe48-11ea-8710-b82a7f25dec1.png)

![image](https://user-images.githubusercontent.com/1621749/94173542-ecabbd80-fe48-11ea-8b65-d35083b73671.png)

**Comment by gari01234** [comment]
the first one is my favorite. It looks great with the Badges!!

**Comment by jonfroehlich** [comment]
Thanks @gari01234. This will be a slightly dev/design TODO for me, so probably won't be done by the time we ship the new Mexico tags and leaderboard.

Like the leaderboard, I'm trying to think of features that will help our mapathons, however.

**Comment by misaugstad** [comment]
I don't think I like the idea of including how frequently others agree... It encourages people to conform to the average, but we don't know that the average is where people should be aiming.

And I agree with @gari01234 that I like the first one the best, but that also requires a system of badges that we don't have yet lol.

And what is the point of surfacing the user's (and other users') agreement percentage? Why does your average user care about that? I think you and I find that interesting, but I feel like most people wouldn't care and it just takes up space. The total number they've done, a progress bar, and badges are the things I expect people to actually enjoy looking at.

**Comment by jonfroehlich** [comment]
I also worried about showing other users' agreement percentages both in terms of its potential to nudge towards conformance and needless information clutter. So, we can cut it. I guess I was trying to give people a sense of how they did compare to general behavior. 

Re: badges. This is relatively easy to implement if we keep it a frontend feature (which is just a matter of doing some conditional checks). I think, longterm, we'll probably want to do something more coherent on the backend so we can display this stuff in the user dashboard, etc.

**Comment by misaugstad** [comment]
>I also worried about showing other users' agreement percentages both in terms of its potential to nudge towards conformance and needless information clutter. So, we can cut it.

Agreed.

>Re: badges. This is relatively easy to implement if we keep it a frontend feature (which is just a matter of doing some conditional checks). I think, longterm, we'll probably want to do something more coherent on the backend so we can display this stuff in the user dashboard, etc.

I think that's fair, especially if the badges are just about total labels validated to start off with (something that we are already keeping track of on the front-end).

**Comment by jonfroehlich** [comment]
I want to revisit this redesign once I'm finished with the new user dashboard designs.

Interestingly, in going back through old UI ideas and mockups for late 2015 and early 2016, I see that we had "Achievements" on the right mission sidebar. Good ideas persist! :D

![image](https://user-images.githubusercontent.com/1621749/94563279-577b4100-021b-11eb-977e-a3ef3cb97993.png)

**Comment by jonfroehlich** [comment]
Iterating on this now that I've been working on the user dashboard (badges, leaderboard trophies, etc.).

This one feels too busy:
![image](https://user-images.githubusercontent.com/1621749/94818583-2170c500-03b3-11eb-81ad-517441bd1188.png)

I would have mouseover (tooltip) text for the overall stats icons:
![image](https://user-images.githubusercontent.com/1621749/94818676-3baaa300-03b3-11eb-9685-93776eb3f4e2.png)

**Comment by jonfroehlich** [comment]
Again, while overseeing Kindergarten, more design work. I think I like this one best so far.

Trying to keep the sidebar clean and minimalistic while still being fun and encouraging.

![image](https://user-images.githubusercontent.com/1621749/94844960-d4511b00-03d3-11eb-9797-3457d22d551f.png)

**Comment by jonfroehlich** [comment]
Newest version.

![image](https://user-images.githubusercontent.com/1621749/95211618-aa1ba680-07a1-11eb-984c-ea8351881a69.png)

**Comment by gari01234** [comment]
Looks great to me! is this sidebar also going to be added to the explore mission screens?

**Comment by jonfroehlich** [comment]
Yes, I'm working on that too but a bit further behind on that. Here's an early draft:

![image](https://user-images.githubusercontent.com/1621749/95460523-eda11c80-0929-11eb-8486-b2027f319b6f.png)

I also like the idea of keeping track of stats and certain achievements **per neighborhood** in addition to overall. But I'm just haven't had much time to mock things up:

![image](https://user-images.githubusercontent.com/1621749/95460694-1fb27e80-092a-11eb-81fc-ae446c110938.png)

**Comment by jonfroehlich** [comment]
Another draft:

![image](https://user-images.githubusercontent.com/1621749/95630471-bc624280-0a36-11eb-8612-ee10a6b2826e.png)

**Comment by jonfroehlich** [comment]
I think the MVP version of this, as discussed with @misaugstad, is simply to add a lil version of the dashboard stats on the sidebar.  A key advantage is always showing the "accuracy" stat (which is currently only available if you're on the leaderboard or if you click on your user dashboard).

![image](https://user-images.githubusercontent.com/1621749/159358915-5c3fe481-962f-4706-9d3f-00ffe3ec2ffa.png)

![image](https://user-images.githubusercontent.com/1621749/159358960-c58246a6-87e1-4ffd-878c-99ae707945db.png)

**Comment by jonfroehlich** [comment]
As this is part of our ongoing UI redesigns, just putting this here from Slack.

If I might take a swing at synthesizing this into higher level themes, we want our UI redesigns to focus on:

**User education:** how can we continuously help users learn about urban design and accessibility assessment. Relatedly, how can we help them become more effective labelers. One great example of this is the mission screen redesigns but another is feedback—how can users get timely feedback on their performance so they can take corrective action. See [Section 1.1.2](https://makeabilitylab.cs.washington.edu/media/publications/Sensing_and_Feedback_of_Everyday_Activities_to_Promote_Environmental_Behaviors_Eye7nYF.pdf) of my PhD dissertation.

**Collective action:** how can we help people feel part of something bigger than themselves. For example, the sidebar should not just track my progress in a neighborhood but the overall progress of our collective efforts together. Being part of a community is a human need, and the best way to make a difference.

**Fun, motivation, engagement.** how can we make the experience feel fun and engaging. What are current pain points (one you identified is the "overwhelm" that users might feel in starting or feelings of "anxiety" when they don't know how to label something). Sometimes, little UI tweaks (like animations) can help make an experience feel more cohesive and fun.
Making a difference. how can/should we remind people of how their work can make a difference?

**Comment by jonfroehlich** [comment]
After seeing some of these mocks from 2020, I'm not as certain about them now! They seem a bit overly dense and complicated. 

Oh, how time heals and changes us.

**Comment by jonfroehlich** [comment]
Here are some new design updates.

## Current implementation
<img width="1825" height="931" alt="Image" src="https://github.com/user-attachments/assets/2595ad8a-faa2-45b5-9f29-eb39fb7f61cc" />

## Jon's riff
Note: there is now an awkward vertical gap that needs to be resolved.

<img width="2823" height="1528" alt="Image" src="https://github.com/user-attachments/assets/fe0a6205-ab72-46b1-a628-94375e60230a" />
